examples/hello-world/hello-service.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+import asyncio
+from datetime import datetime, timedelta
+
+# from yapapi.log import enable_default_logger, log_summary, log_event_repr  # noqa
+
+from yapapi import Golem
+from yapapi.executor.services import Service
+from yapapi.payload import vm
+
+DATE_OUTPUT_PATH = "/golem/work/date.txt"
+POLLING_INTERVAL_SEC = 5
+
+
+class DateService(Service):
+    @staticmethod
+    async def get_payload():
+        return await vm.repo(
+            image_hash="d646d7b93083d817846c2ae5c62c72ca0507782385a2e29291a3d376",
+        )
+
+    async def start(self):
+        self._ctx.run(
+            "/bin/sh",
+            "-c",
+            f"while true; do date > {DATE_OUTPUT_PATH}; sleep {POLLING_INTERVAL_SEC}; done &",
```

Comment:
I'd differentiate the interval that's used for _polling_ of the service, from the one used to update the file itself... 

unsure about the name though

```suggestion
            f"while true; do date > {DATE_OUTPUT_PATH}; sleep {TICK_INTERVAL}; done &",
```

Reply:
Updated in 5807d28e201847681fe2ff3e0c336e06fb512211